### DISCUSSION TITLE
run error problem

### DISCUSSION THREAD

**Discussion by lv333ming** [python-poetry/poetry#7565] [General]
![poetry](https://user-images.githubusercontent.com/12588535/221387946-5f74c281-5709-4510-a7eb-05f7acc6f0b5.jpg)
I don’t know how to deal with the running error

**Comment by PabloEmidio** [comment]
It does not look like a poetry related problem. Could you give more informations about your scenario to help understand what is causing the error?

**Comment by ralbertazzi** [comment]
Closed as stale and not relevant to Poetry
